content/python/tags.py.md

```diff
@@ -0,0 +1,215 @@
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Get the tags attached to references.
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+tag_service = client['Tag']
+
+try:
+    response = tag_service.getAttachedTagsForCurrentUser()
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Retrieve a SoftLayer_Tag record.
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+tag_service = client['Tag']
+
+tag_id = 123456
+
+try:
+    response = tag_service.getObject(id=tag_id)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Retrieve references that tie object to the tag.
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+tag_service = client['Tag']
+
+tag_id = 123456
+
+try:
+    response = tag_service.getReferences(id = tag_id)
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Get the tag object based on what the user inputs.
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+tag_service = client['Tag']
+
+try:
+    response = tag_service.getTagByTagName()
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Get the tags not attached to references.
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+tag_service = client['Tag']
+
+try:
+    response = tag_service.getUnattachedTagsForCurrentUser()
+    print(json.dumps(response, sort_keys=True, indent=2, separators=(',', ': ')))
+except SoftLayer.SoftLayerAPIError as e:
+    print("Unable to list the response for the package: \nfaultCode= %s, \n \nfaultString= %s"
+          % (e.faultCode, e.faultString))
+```
+
+### Set the tags for a given object.
+
+```python
+import SoftLayer
+import json
+
+client = SoftLayer.create_client_from_env()
+tag_service = client['Tag']
+
+tag_name = 'tag example'
+tag_type = 'ACCOUNT_DOCUMENT'
```

Comment:
Use `GUEST` or `HARDWARE` as an example. I'm honestly not even sure what an `ACCOUNT_DOCUMENT` refers to...